configs/mender_convert_config

```diff
@@ -124,9 +124,11 @@ MENDER_CLIENT_INSTALL="y"
 #
 # This is used to fetch the correct binaries
 #
-# Valid values are "latest" (default), "master" or a specific version
-# MEN-6948 mender-convert is not yet integrated with latest mender-client 4.0.0
-MENDER_CLIENT_VERSION="3.5.2"
+# Valid values are "auto" (default), "latest", "master" or a specific version
+# If set to "auto" then the Mender Client 4.0 or later will be installed unless
+# the distribution is Ubuntu jammy or older or Debian bullseye or older;
+# the latest Mender 3.x client will be installed in those cases.
+MENDER_CLIENT_VERSION="auto"
```

Comment:
Okey, and what "latest" will mean in this context? We should document it

Reply:
Independently, we need to update [this docs page](https://docs.mender.io/operating-system-updates-debian-family/variables#mender_client_version).

Reply:
Latest is the latest version, so 4.0.2 as of today. 
I will follow up with the documentation.